package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "gpii-pouch",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "test": "node src/tests/all-tests.js"
+  },
+  "dependencies": {
+    "express":         "4.10.5",
+    "express-pouchdb": "0.14.0",
+    "infusion":        "git://github.com/fluid-project/infusion.git#df5f8cfabf815a4086b778e73125c3c952dda4ec",
+    "memdown":         "1.0.0",
+    "pouchdb":         "3.0.5",
```

Comment:
Apparently if you update to the latest version (3.3.1 at time of writing), leveldb is now an "optionalDependency" which means that this module will install on Windows even if the build of it fails (which it will without the bizarre incantations)